serverless/sql-databases/reference-content/serverless-sql-databases-overview.mdx

```diff
@@ -38,10 +35,19 @@ Autoscaling triggers automatically based on your database vCPU consumption:
 
 \*\* The number of maximum connections scales with the allocated vCPUs and RAM, and will increase with upcoming releases. 
 
-## Idle state
-After receiving no query for 5 minutes, a Serverless SQL Database will enter into an idle state. It switches from **Running** to **Ready to use**.
+## Autoscaling
 
-While your database is in an idle state, you will not be billed for compute resources (vCPU and RAM). You will however still be billed for storage resources.
+Autoscaling triggers automatically based on your database vCPU consumption:
+
+  - **Scaling up**: after using more than 40% of the currently allocated vCPU for 10 seconds, the vCPU and RAM allocated to your database will increase by 1 vCPU and 4 GB RAM each time, up to a maximum of 15 vCPUs and 60 GB RAM.
+
+  - **Scaling down**: after using less than 2% of the currently allocated vCPU for 10 seconds, the vCPU and RAM allocated to your database will decrease by 1 vCPU and 4 GB RAM each time, down to a minimum of 1 vCPU and 4 GB RAM while your database is active.
+  
+<Message type="important">
+    These values may change over time to optimize the efficiency of automatic scaling operations.
+  </Message>
+
+Automatic scaling operations only happen **between** atomic transactions. For example, a long-running transaction (such as updating millions of rows) taking one minute will delay the next scaling operation by one minute. The next transaction performed will, however, benefit from the upscaled/downscaled database performance.
```

Comment:
@fpagny This part is still "to assess"

Reply:
No it has been cleared with engineering, although it's not deployed into production yet. This part will be true only in March when deployment is done.